### PR TITLE
Updated Docker Guidance

### PR DESCRIPTION
**Pull request by lchapman4**
Updated the Docker log forwarding options.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/logs/forward-logs/enable-log-management-new-relic.mdx [hunk 1]
```diff
@@ -75,7 +75,20 @@ Here are some details on how to accomplish specific use cases:
...
+        * Forward Docker logs using our infrastructure agent. In the config file be sure to tail the `/var/log/containers/*.log` directory.
+          **Example:**
+
+          ```yml
+          logs:
+           - name: Docker-logs
+             file: /var/log/containers/*.log # Path to Docker log files
+          ```
+        * Use our [Standalone Docker image](https://hub.docker.com/r/newrelic/newrelic-fluentbit-output).
+
```
Comment (ally-sassman) [inline src/content/docs/logs/forward-logs/enable-log-management-new-relic.mdx:91]:
```suggestion
        For Kubernetes environments, use our [Kubernetes plugin](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/).
        
        For Docker environments, we offer a few options:
        * Use our [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context)
        * Use our infrastructure agent. In the `logging.yml` config file be sure to tail the `/var/log/containers/*.log` directory.
          **Example:**

          ```yml
          logs:
           - name: Docker-logs
             file: /var/log/containers/*.log # Path to Docker log files
          ```
        * Use our [Standalone Docker image](https://hub.docker.com/r/newrelic/newrelic-fluentbit-output).

```

File: src/content/docs/logs/forward-logs/enable-log-management-new-relic.mdx [hunk 2]
```diff
@@ -75,7 +75,18 @@ Here are some details on how to accomplish specific use cases:
         Collect logs from apps running in Docker containers, either standalone or when using Kubernetes
       </td>
       <td>
-        Use our [Kubernetes plugin](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/) or our [infrastructure monitoring agent with Docker](/docs/infrastructure/install-infrastructure-agent/linux-installation/docker-container-infrastructure-monitoring/#requirements).
+        For Kubernetes environments, use our [Kubernetes plugin](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/).
+        
+        For Docker environments, we offer a few options:
+        * Use our [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context)
```
Comment (ally-sassman) [inline src/content/docs/logs/forward-logs/enable-log-management-new-relic.mdx:81]: @lchapman4 For the logs in context option, do they need to do anything additional for Docker? Does that mean that all Docker customers will automatically have their host logs collected _because_ they're using APM agents?

Comment (lchapman4) [reply src/content/docs/logs/forward-logs/enable-log-management-new-relic.mdx:81]: For the apm logs-in-context option nothing else is needed on their part. That said, if they use the apm method they will not collect host-level logs. That's a natural limitation of docker. To collect host level logs they would need to use our infra agent installed on the host.

Comment (lchapman4) [reply src/content/docs/logs/forward-logs/enable-log-management-new-relic.mdx:81]: Hey @ally-sassman sorry, I lost track of this. Is anything else needed for this PR?

File: src/content/docs/logs/forward-logs/enable-log-management-new-relic.mdx [hunk 3]
```diff
@@ -75,7 +75,18 @@ Here are some details on how to accomplish specific use cases:
...
+        * Use our infrastructure agent. In the `logging.yml` config file be sure to tail the `/var/log/containers/*.log` directory. For example:
+          ```yml
+          logs:
+           - name: Docker-logs
+             file: /var/log/containers/*.log # Path to Docker log files
+          ```
+        * Use our [Standalone Docker image](https://hub.docker.com/r/newrelic/newrelic-fluentbit-output).
```
Comment (ally-sassman) [inline src/content/docs/logs/forward-logs/enable-log-management-new-relic.mdx:88]:
```suggestion
        * Use our [Standalone Docker image](https://hub.docker.com/r/newrelic/newrelic-fluentbit-output)
```

File: src/content/docs/logs/forward-logs/enable-log-management-new-relic.mdx [hunk 4]
```diff
@@ -75,7 +75,18 @@ Here are some details on how to accomplish specific use cases:
         Collect logs from apps running in Docker containers, either standalone or when using Kubernetes
       </td>
       <td>
-        Use our [Kubernetes plugin](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/) or our [infrastructure monitoring agent with Docker](/docs/infrastructure/install-infrastructure-agent/linux-installation/docker-container-infrastructure-monitoring/#requirements).
+        For Kubernetes environments, use our [Kubernetes plugin](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/).
+        
+        For Docker environments, we offer a few options:
+        * Use our [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context)
+        * Use our infrastructure agent. In the `logging.yml` config file be sure to tail the `/var/log/containers/*.log` directory. For example:
```
Comment (ally-sassman) [inline src/content/docs/logs/forward-logs/enable-log-management-new-relic.mdx:82]:
```suggestion
        * Use our infrastructure agent to collect host-level logs. In the `logging.yml` config file be sure to tail the `/var/log/containers/*.log` directory. For example:
```

**Review by ghost — APPROVED** [review]

**Pull request merged**